cli/linter/schema.json

```diff
@@ -448,13 +448,25 @@
         "type": "string",
         "enum": [
           "",
+          "bcrypt",
```

Comment:
bcypt not needed here, e.g. we should not has key IDs using it.

Reply:
Also based on the code, we have not modified this part too, so just needs be removed from here ,and from the code comment below

Reply:
Removed! :+1: